src/docs/release/breaking-changes/add-showAutocorrectionPromptRect.md

```diff
@@ -0,0 +1,90 @@
+---
+title: New method `showAutocorrectionPromptRect` added to `TextInputClient` 
+description: a new method `void showAutocorrectionPromptRect(int start, int end)` was added to the `TextInputClient` interface
+---
+
+
+## Summary
+
+A new method `void showAutocorrectionPromptRect(int start, int end)` was added to the `TextInputClient` interface.
+
+## Context
+
+In order to display the iOS autocorrection highlight, the iOS text input plugin needed a way to inform the Flutter framework of 
+the highlight's start and end position.
+
+## Description of change
+
+A new method `void showAutocorrectionPromptRect(int start, int end)` was added to the `TextInputClient` interface. This method will
+be called by iOS when it finds a new potential autocorrect candidate in the current user input, or when the range of a previously highlighted candidate changes.
+
+## Migration guide
+
+If your application does not implement or subclass `TextInputClient`, no migration is needed.
+If your application does not target iOS, or the class that implemented the `textInputClient` interface does not 
+support autocorrect, you only need to add an empty implementation for the new method:
+
+```dart
+class CustomTextInputClient implements TextInputClient {
+  void showAutocorrectionPromptRect(int start, int end) {
+  }
+}
+```
+
+Otherwise, if your app targets iOS and supports autocorrect on iOS, it is recommended to add a sensible implementation of `void showAutocorrectionPromptRect(int start, int end)` 
+to your `TextInputClient` subclass. 
+
+Code after migration:
+
+<!-- skip -->
```

Comment:
why skip?

Reply:
we should make this valid code that we can analyze, so that we continue to update it

Reply:
The code doesn't compile: `class CustomTextInputClient extends State<...> implements TextInputClient`